### DISCUSSION TITLE
What are the Mandatory packages?

### DISCUSSION THREAD

**Discussion by rajeevrrs** [modoboa/modoboa#2678] [Installer]
Hi, 

Modoboa Install 14 packages which makes my 1 GB RAM + 1 GB CPU server heavy load and several time crash. 

1. modoboa
2. automx
3. amavis
4. clamav
5. dovecot
6. nginx
7. razor
8. postfix
9. postwhite
10. spamassassin
11. uwsgi
12. radicale
13. opendkim
14. SQL (MySQL/postgresql)

I Want : 

1. Dovecot - IMAP & POP for receiving mail 
2. modoboa UI - to create, manage & delete identities and alias

Now, the question is what are the mandatory dependencies package to support my above mentioned two operation (Eg MySQL, Nginx, etc?). So, i can avoid rest to install

**Comment by Spitfireap** [comment]
Hi,

You can disable any install by setting enabled to false for every section you don't want. For what you want, it seems you'll need : 
``modoboa dovecot nginx postfix (postwhite) uwsgi opendkim SQL`` Basically with 1GB memory you can forget amavis. It should run well

**Comment by rajeevrrs** [comment]
@Spitfireap 

I do not want Postfix (SMTP) and also i do not want Modoboa webmail. I guess Postwhite, uwsgi and opendkim is used for STMP. 

All i need two operation. To support these two operation what are the additional package i have to install?

    Dovecot - IMAP & POP for receiving mail
    modoboa UI - to create, manage & delete identities and alias

**Reply by Spitfireap** [reply]
I don't see how you could receive email without SMTP :)
Else, everything you want can be customized on installer.cfg file. 
uwsgi is not used for SMTP, but for the website.

**Comment by rajeevrrs** [comment]
@Spitfireap 

SMTP main function is to send mail. IMAP & POP is used to receive mail. I will be using external SMTP Services. 

So, as per you `Dovecot + modoboa + uwsgi + nginx + SQL + SSL = works fine? `

**Reply by Spitfireap** [reply]
I believe SMTP is also for receiving mail and IMAP and POP is for storing/distributing mail... Anyway, keep in mind that modoboa has not been developed for such use case, so there is no guarantee that it will indeed work, best is to test it :D

**Reply by rajeevrrs** [reply]
Got it. Let me test.

**Comment by rajeevrrs** [comment]
@tonioo 

`Dovecot + modoboa + uwsgi + nginx + SQL + SSL = works fine`? for

```
Dovecot - IMAP & POP for receiving mail
modoboa UI - to create, manage & delete identities and alias

```

**Reply by tonioo** [reply]
What's the point in disabling postfix? You won't receive any mail if you do that.

**Reply by rajeevrrs** [reply]
@tonioo i thought Postfix is only for SMTP. i will be using Amazon SES for SMTP

`Dovecot + Posfix + modoboa + uwsgi + nginx + SQL + SSL = Now it will work?` or i need postwhite & opendkim?
